sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/blob/perf/DownloadBlobNonSharedClientTest.java

```diff
@@ -13,44 +13,52 @@
 import java.io.OutputStream;
 
 public class DownloadBlobNonSharedClientTest extends AbstractDownloadTest<BlobPerfStressOptions> {
-    private static final int BUFFER_SIZE = 16 * 1024 * 1024;
-    private static final OutputStream DEV_NULL = new NullOutputStream();
+    private static final int DEFAULT_BUFFER_SIZE = 16 * 1024 * 1024;
```

Comment:
```suggestion
    private static final int DEFAULT_BUFFER_SIZE = 4 * 1024 * 1024;
```

As discussed offline, we think this should be reduced to 4MB (or even lower) to reduce memory usage during tests.  If we do lower this, we should verify it doesn't impact throughput on the 10MB or 1GB tests.